scripts/west_commands/runners/canopen_program.py

```diff
@@ -262,11 +270,11 @@ def download(self, bin_file):
         try:
             size = os.path.getsize(bin_file)
             infile = open(bin_file, 'rb')
-            outfile = self.data_sdo.open('wb', size=size)
+            outfile = self.data_sdo.open('wb', size=size, block_transfer=self.block_transfer)
 
             progress = Bar('%(percent)d%%', max=size, suffix='%(index)d/%(max)dB')
             while True:
-                chunk = infile.read(1024)
+                chunk = infile.read(256)
```

Comment:
??

Reply:
Thanks. I have replaced the magic number with named constants and added a description to the Git commit log.